b2sdk/raw_api.py

```diff
@@ -290,7 +302,8 @@ def _post_json(self, base_url, api_name, auth, **params):
         :param auth: passed in Authorization header
         :param api_name: example: "b2_create_bucket"
         :param args: the rest of the parameters are passed to b2
-        :return:
+        :return: the decoded JSON response
+        :rtype: dict
```

Comment:
` -> Dict[str, Any]:` in the method signature should be better (some IDEs support it etc)